modules/gst/gst.c

```diff
@@ -147,12 +144,6 @@ static void format_check(struct ausrc_st *st, GstStructure *s)
 		warning("gst: expected %d channels (got %d)\n",
 			st->prm.ch, channels);
 	}
-	if (16 != width) {
```

Comment:
Not needed anymore. The audio format must be S16LE as defined in set_caps function.